mscore/mu3paletteadapter.h

```diff
@@ -0,0 +1,52 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU3PALETTEGETACTION_H
+#define MU3PALETTEGETACTION_H
+
+#include "mu4/scenes/palette/ipaletteadapter.h"
+
+namespace Ms {
```

Comment:
Why still `Ms` for this one?

Reply:
This class refers to the "old code". As soon as MU 4 is more or less ready, we will make a decision and move on to MU 4, and the classes related to MU 3, like this one, MuseScore, ScoreView, etc. will be deleted.